openeogeotrellis/job_tracker.py

```diff
@@ -345,12 +376,13 @@ def _to_serializable_datetime(epoch_millis: str) -> Union[str, None]:
 
     parser = argparse.ArgumentParser(usage="OpenEO JobTracker", formatter_class=argparse.ArgumentDefaultsHelpFormatter)
     parser.add_argument("--principal", default="openeo@VGT.VITO.BE", help="Principal to be used to login to KDC")
-    parser.add_argument("--keytab", default="openeo-deploy/mep/openeo.keytab",
+    parser.add_argument("--keytab", default="openeo.keytab",
                         help="The full path to the file that contains the keytab for the principal")
     args = parser.parse_args()
 
     try:
-        JobTracker(ZkJobRegistry, args.principal, args.keytab).update_statuses()
+        vault = Vault("https://vault.vgt.vito.be")
```

Comment:
Add a default constructor argument?

Reply:
in my feature branch I was currently going for
```
Vault(ConfigParams().vault_url)
```
(14d7c7abfc01c0060f33a902dda58aa897c8192b)